README.md

```diff
@@ -30,7 +30,7 @@ most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
 
 ## Good Invariant Reasoning:
-My code's invariant is that, at the start of each recursive call to fib(n), the function is designed to return an array that perfectly contains the Fibonacci sequence up to and including the nth element (in alignment with zero-based indexing). Thus ensuring that the 'state of the world' at each step of the recursion accurately builds upon the previous steps, which maintains the integrity of the sequence throughout. <br />
+My code's invariant is that, at the start of each recursive call to fib(n), the function computes the Fibonacci sequence up to the nth term and returns it as an array. Each element in the array represents a term in the Fibonacci sequence, beginning from the first term (0) and including up to the nth term, with the sequence following a zero-based indexing set-up. This process involves recursively calling the function to build the sequence one by one, ensuring the sequence is accurate and complete for any given n. Thus guaranteeing that the 'state of the world' at each step of the recursion accurately builds upon the previous steps, which maintains the integrity of the sequence throughout. <br />
```

Comment:
What function computes the Fibonacci sequence up to the requested term? Your function? This sounds like you're saying that the invariant is that the function works...